spec/views/cases/_case_status_html_slim_spec.rb

```diff
@@ -68,7 +70,8 @@
       current_state: 'drafting',
       type_abbreviation: 'FOI',
       who_its_with: 'DACU',
-      type_of_offender_sar?: false
+      type_of_offender_sar?: false,
```

Comment:
Why have both of these methods? They seem to essentially be duplicates of each other.

Reply:
offender_sar? is standard ones, type_of_offender_sar? is either standard or complaint